.env.production

```diff
@@ -7,7 +7,7 @@ REACT_APP_LIBRARY_BACKEND=https://us-central1-excalidraw-room-persistence.cloudf
 REACT_APP_PORTAL_URL=https://portal.excalidraw.com
 # Fill to set socket server URL used for collaboration.
 # Meant for forks only: excalidraw.com uses custom REACT_APP_PORTAL_URL flow
-REACT_APP_WS_SERVER_URL=
+REACT_APP_WS_SERVER_URL=http://localhost:3002
```

Comment:
?

Reply:
again :)

You should create a `.env.local`

Reply:
oops! yep will do :)